src/log.c

```diff
@@ -40,6 +42,9 @@ struct log_param_s {
 	int syslog_prio;
 };
 
+static int initial_stdout;
```

Comment:
Can anyone explain what this (and the other related changes below) is supposed to fix?